### PR TITLE
FEAT: Added bulk operation in database file and updated output format in request processor

### PR DESCRIPTION
**Pull request by shehram-tahir**

### CODE REVIEW

File: dtypes/response_dtypes.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+from pydantic import BaseModel
+
+
+T = TypeVar("T")
+
+
+class ResponseModel(BaseModel, Generic[T]):
+    message: str
+    responseCode: int
+    data: T
```
Comment (abdullahalhoothy) [inline dtypes/response_dtypes.py:12]: Why is this still here .I thought we agreed to remove it

**Review by abdullahalhoothy — CHANGES_REQUESTED** [review]

File: dtypes/response_dtypes.py (hunk 1 above)
Comment (shehram-tahir) [reply dtypes/response_dtypes.py:12]: We might need to use this validation, so I have not removed it. But I have removed it's usage, so it is not in use anymore

File: request_processor.py [hunk 2]
```diff
@@ -37,6 +37,4 @@ async def request_handling(
                 status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
                 detail=f"An unexpected error occurred: {str(e)}",
             ) from e
-
-    res_body = output_type(**output)
-    return res_body
+    return output
```
Comment (abdullahalhoothy) [inline request_processor.py:40]:
```suggestion

    res_body = output_type(**output)
    return res_body
```

File: request_processor.py [hunk 3]
```diff
@@ -17,7 +18,6 @@
 async def request_handling(
     req: Optional[T],
     input_type: Optional[Type[T]],
```
Comment (abdullahalhoothy) [inline request_processor.py:20]:
```suggestion
    input_type: Optional[Type[T]],
    output_type: Type[U],
```

**Review by abdullahalhoothy — CHANGES_REQUESTED** [review]

**Review by abdullahalhoothy — APPROVED** [review]

**Pull request merged**